src/lib/adapters/transaction.ts

```diff
@@ -88,7 +116,7 @@ const sepoliaBlockchain: BlockchainNetwork = {
 	},
 }
 
-export const defaultBlockchainNetwork = chiadoBlockchain
+export const defaultBlockchainNetwork = gnosisBlockchain
```

Comment:
Can we make this env variable? So something like
```suggestion
export const defaultBlockchainNetwork = blockchains[PUBLIC_ENV_CHAIN_NAME] ?? testnet
```

Reply:
I don't mind making this configurable with an env variable, but I would make the gnosis the default, otherwise you must configure it everywhere where we test (locally, on vercel, etc.)

Reply:
Not going to make this change in this PR, but I opened an issue about it: https://github.com/logos-innovation-lab/waku-objects-playground/issues/452